app/api/init_api.py

```diff
@@ -131,6 +132,14 @@ async def on_startup() -> None:
         await app.state.services.database.connect()
         await app.state.services.redis.initialize()
 
+        if app.settings.RABBITMQ_ENABLED:
+            app.state.services.amqp = await aio_pika.connect_robust(
+                host=app.settings.RABBITMQ_HOST,
+                port=app.settings.RABBITMQ_PORT,
+            )
+
+        app.state.services.amqp_channel = await app.state.services.amqp.channel()
```

Comment:
should this be indented? looks like an err with `app.state.services.amqp` undefined otherwise

Reply:
yup, you're right